pandas/core/generic.py

```diff
@@ -2055,7 +2055,6 @@ def __setstate__(self, state) -> None:
                 object.__setattr__(self, "_attrs", attrs)
                 flags = state.get("_flags", {"allows_duplicate_labels": True})
                 object.__setattr__(self, "_flags", Flags(self, **flags))
-
```

Comment:
Can you revert this line removal. Shouldn't have any diff in this file.

Reply:
resolved